openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -113,36 +111,65 @@ def update_op_assets(
         except (TypeError, ValueError):
             frame_in = 1001
         item_data["frameStart"] = frame_in
-        # Frames duration, fallback on 0
+        # Frames duration, fallback on 1
         try:
             # NOTE nb_frames is stored directly in item
             # because of zou's legacy design
-            frames_duration = int(item.get("nb_frames", 0))
+            frames_duration = int(item.get("nb_frames", 1))
         except (TypeError, ValueError):
-            frames_duration = 0
+            frames_duration = None
         # Frame out, fallback on frame_in + duration or project's value or 1001
         frame_out = item_data.pop("frame_out", None)
         if not frame_out:
-            frame_out = frame_in + frames_duration
-        try:
-            frame_out = int(frame_out)
-        except (TypeError, ValueError):
-            frame_out = 1001
+            if frames_duration:
+                frame_out = frame_in + frames_duration - 1
+            else:
+                frame_out = project_doc["data"].get("frameEnd", 1001)
```

Comment:
Should this default to `frame_in` instead of hardcoded `1001`?
```suggestion
                frame_out = project_doc["data"].get("frameEnd", frame_in)
```

Reply:
Yeah that's better